COR:Correct magic number

cherry2030 opened the pull request:
Correct magic number to constant.
Declare new constant like below.

const int gotoGame = 1;
const int gotoLeaderBoard = 2;
const int quitGame = 3;

const int someoneWin = 1;
const int keepGoing = 0;

const int horizontal_complete = 1;
const int horizontal_fail = 0;

const int vertical_complete = 1;
const int vertical_fail =0;

const int diagonal_complete = 1;
const int diagonal_fail = 0;

const int draw_complete =-1;
const int draw_fail = 0;

Commit:
COR:Correct magic number

Correct magic number to constant.
Declare new constant like below.

const int gotoGame = 1;
const int gotoLeaderBoard = 2;
const int quitGame = 3;

const int someoneWin = 1;
const int keepGoing = 0;

const int horizontal_complete = 1;
const int horizontal_fail = 0;

const int vertical_complete = 1;
const int vertical_fail =0;

const int diagonal_complete = 1;
const int diagonal_fail = 0;

const int draw_complete =-1;
const int draw_fail = 0;

mjseok commented:
current_player에대한 매직넘버도 없애는게 더 좋을 것 같아요~! current_player=2 대신 current_player= player2이런식으로 바꾸는거 어떨까요?

seohyeonKK commented:
> current_player에대한 매직넘버도 없애는게 더 좋을 것 같아요~! current_player=2 대신 current_player= player2이런식으로 바꾸는거 어떨까요?

민지님이 언급하신 거 동의합니다~

Commit:
Declare new constant like below.

const int player1_turn =1;
const int player2_turn =2;
Change magic number to this constant.

hanzzb commented:
좋아요 머지합시당 ~~
